### DISCUSSION TITLE
Add History Feature

### DISCUSSION THREAD

**Discussion by pavelschroeder** [logancyang/obsidian-copilot#202] [Ideas]
An important feature would be the ability to view chat history within each note, allowing for easy reference and organization.

**Comment by logancyang** [comment]
Thanks for your suggestion @pavelschroeder ! Not sure if I understand correctly. Do you mean that each note should have a list of old chats that used this note as context?

I'm going to add support for `QA: multiple notes` mode, in that case, what should this history anchor on?
